src/masternodes/vault.h

```diff
@@ -0,0 +1,64 @@
+#ifndef DEFI_MASTERNODES_VAULT_H
+#define DEFI_MASTERNODES_VAULT_H
+
+#include <amount.h>
+#include <uint256.h>
+
+#include <flushablestorage.h>
+#include <masternodes/balances.h>
+#include <masternodes/res.h>
+#include <script/script.h>
+
+// use vault's creation tx for ID
+using CVaultId = uint256;
+
+struct CVaultMessage {
```

Comment:
Why this struct not name as `CCreateVaultMessage`?

Reply:
It is following convention in Loan but could be changed for better understanding of purpose